### PR TITLE
Try adding alert to bad-plugin-value

### PR DESCRIPTION
**Pull request by ollie-iterators**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Working on adding an alert for bad-plugin-value

NOTE: This one is strange because it seems this error deals with pylintrc.

Refs #5953 

Closes #XXXX

### CODE REVIEW

File: doc/data/messages/b/bad-plugin-value/good.py [hunk 1]
```diff
@@ -1 +1 @@
-# This is a placeholder for correct code for this message.
+# This issue happens with pylintrc
```
Comment (mbyrnepr2) [inline doc/data/messages/b/bad-plugin-value/good.py:1]: Is it strictly necessary to include this file?

**Review by mbyrnepr2 — APPROVED** [review]
Looks pretty good! One remark.

**Comment by DanielNoord** [comment]
See https://pylint--8216.org.readthedocs.build/en/8216/user_guide/messages/error/bad-plugin-value.html

This isn't really helping anybody. We need to change the way these messages render for `pylintrc` related messages to show up properly.

**Comment by ollie-iterators** [comment]
Ok, closing it

**Pull request closed without merging**